### PR TITLE
A Reviewing section

### PR DESCRIPTION
**Pull request by SiccarPoint**
This in-progress PR partly addresses https://github.com/alan-turing-institute/the-turing-way/issues/372, by adding some text expanding on reviewing processes. This needs to go up in advance of the conclusion of our SSI Collaborations Workshop code dash.

I'm hoping to work a bit more on this on the train later, but early comments very welcome (tell me to stop as necessary!), and if anyone feels keen to carry on, feel free...

### CODE REVIEW

**Comment by SiccarPoint** [comment]
I think this is now 95% complete, content-wise. However, I've really been struggling to find appropriate open-license references for this material (so those sections remain `lorem ipsum`-ed). This might be a good time for a better established editor to step in, who knows where the good quality open resources for this kind of thing can be found...

**Comment by rainsworth** [comment]
Thank you so much for this contribution @SiccarPoint! We are working on getting this reviewed and merged 👍

File: content/reproducible_environments/reproducible_environments.md [hunk 1]
```diff
@@ -935,7 +935,7 @@ Hopefully you will give your volume a more descriptive name than volume_name. A
 <a name="What_to_learn_next"></a>
 ## What to learn next
 
-We recommend reading the chapter on testing, and then the chapter on continuous integration. Note that the chapter on version control is a prerequisite for the chapter on continuous integration. The open research chapter also contains further information on sharing research respoducibly.
+We recommend reading the chapter on testing, and then the chapter on continuous integration. Note that the chapter on version control is a prerequisite for the chapter on continuous integration. The open research chapter also contains further information on sharing research reproducibly.
```
Comment (rainsworth) [inline content/reproducible_environments/reproducible_environments.md:938]: 👍

**Review by rainsworth — APPROVED** [review]
Hi @SiccarPoint - This is brilliant and I'm going to get it merged in. We are in the process of incorporating the book repo back into the main Turing Way repo, so would you mind adding anything that's left to do on this chapter in your original issue https://github.com/alan-turing-institute/the-turing-way/issues/372?

For example, add:
> I think this is now 95% complete, content-wise. However, I've really been struggling to find appropriate open-license references for this material (so those sections remain `lorem ipsum`-ed). This might be a good time for a better established editor to step in, who knows where the good quality open resources for this kind of thing can be found...

Thank you for all your contributions to the Turing Way! 🎉

**Comment by SiccarPoint** [comment]
Sure!

**Comment by illushka** [comment]
Okay great. I made a pull request with my suggested changes: https://github.com/alan-turing-institute/the-turing-way-book/pull/27

**Pull request merged**